packages/@react-spectrum/tag/src/Tag.tsx

```diff
@@ -72,9 +71,9 @@ export function Tag<T>(props: SpectrumTagProps<T>) {
             text: {UNSAFE_className: classNames(styles, 'spectrum-Tag-content'), ...labelProps},
             avatar: {UNSAFE_className: classNames(styles, 'spectrum-Tag-avatar'), size: 'avatar-size-50'}
           }}>
-          {typeof children === 'string' ? <Text>{children}</Text> : children}
+          {typeof item.rendered === 'string' ? <Text>{item.rendered}</Text> : item.rendered}
```

Comment:
changing the types shouldn't have changed this, was it just wrong before but still somehow worked?

Reply:
`children` was in `ItemProps` which `TagProps` extended. Technically worked, but doesn't match our other components.